mail_composer_cc_bcc/tests/test_mail_cc_bcc.py

```diff
@@ -55,12 +56,11 @@ def test_upstream_file_hash(self):
         """Test that copied upstream function hasn't received fixes"""
         func = inspect.getsource(upstream._send).encode()
         func_hash = hashlib.md5(func).hexdigest()
-        self.assertIn(
-            func_hash,
-            VALID_HASHES,
-            "mail.mail#_send has changed in upstream, "
-            "please adapt the override and add the new hash above",
-        )
+        if func_hash not in VALID_HASHES:
+            logging.error(
```

Comment:
Why an error instead of a warning?

Reply:
it's more visible in the log and still doesn't hold up anything